### PR TITLE
Completing implementation of HappyBase connection pool.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: gcloud/bigtable/happybase/pool.py [hunk 1]
```diff
@@ -27,6 +28,14 @@
 """Minimum allowable size of a connection pool."""
 
 
+class NoConnectionsAvailable(RuntimeError):
```
Comment (jonparrott) [inline gcloud/bigtable/happybase/pool.py]: Should this be surfaced in `gcloud.exceptions`?

Comment (dhermes) [inline gcloud/bigtable/happybase/pool.py]: In this case it's part of the public surface of HappyBase. I can implement it in `gcloud.exceptions` and import it here, but I don't feel like there is a payoff there.

Comment (jonparrott) [inline gcloud/bigtable/happybase/pool.py]: That's fine, just wondering.

**Comment by jonparrott** [comment]
LGTM

**Pull request merged**